### PR TITLE
swss: Fix Invalid port oid messages generated because of voq counters.

### PR DESCRIPTION
**Pull request by skbarista**
**What I did**

* Do not add Voq oids to CounterCheckOrch since CounterCheckOrch is not relevant for voqs. Fixed https://github.com/sonic-net/sonic-buildimage/issues/13662 where voq counters stopped working after https://github.com/sonic-net/sonic-swss/pull/2525

**Why I did it**

To fix the invalid oids messages generated by CounterCheckOrch for voq.

**How I verified it**

Verified that the invalid port oid messages go away. Verified voq counters.

### CODE REVIEW

**Comment by vmittal-msft** [comment]
@skbarista Can you please share queuestat output from chassis as well as ToR system to make sure it is working on both ? Also, Could we check if syslogs are clean for both chassis as well as ToR systems ?

**Comment by skbarista** [comment]
[syslog.txt](https://github.com/sonic-net/sonic-swss/files/10759208/syslog.txt)
[tor_counters.txt](https://github.com/sonic-net/sonic-swss/files/10759210/tor_counters.txt)
[voq_stat.txt](https://github.com/sonic-net/sonic-swss/files/10759211/voq_stat.txt)

**Comment by skbarista** [comment]
> @skbarista Can you please share queuestat output from chassis as well as ToR system to make sure it is working on both ? Also, Could we check if syslogs are clean for both chassis as well as ToR systems ?

@vmittal-msft attached the requested files.

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -6163,6 +6163,10 @@ void PortsOrch::generateQueueMapPerPort(const Port& port, FlexCounterQueueStates
         queueVector.emplace_back(name.str(), id);
         if (voq)
         {
+            // Install a flex counter for this voq to track stats. Voq counters do
+            // not have buffer queue config. So it does not get enabled through the
```
Comment (vmittal-msft) [inline orchagent/portsorch.cpp:6167]: FYI, We do apply egress_lossy_profile/egress_lossless_profile on VoQs via SONIC config. This change is there in master branch.

**Review by vmittal-msft — APPROVED** [review]

**Review by vmittal-msft — CHANGES_REQUESTED** [review]
Could you check what these errors are - 

Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x200051500000001 does not has supported counters
Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x201051500000001 does not has supported counters
Feb 16 16:37:50.544500 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x202051500000001 does not has supported counters
Feb 16 16:37:50.544544 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x203051500000001 does not has supported counters
Feb 16 16:37:50.544633 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x204051500000001 does not has supported counters

Also, if we fix the following, Do you think we can fix VoQ counters ? without always enabling them ?

counters
Feb 16 16:37:50.655990 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-8|Asic0|Ethernet64|5-6]
Feb 16 16:37:50.655990 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-5|Asic0|Ethernet24|5-6]
Feb 16 16:37:50.655990 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-8|Asic0|Ethernet108|0-2]
Feb 16 16:37:50.656123 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-5|Asic0|Ethernet172|5-6]
Feb 16 16:37:50.656123 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-8|Asic0|Ethernet20|0-2]
Feb 16 16:37:50.656123 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-8|Asic0|Ethernet168|0-2]
Feb 16 16:37:50.656123 nfc421-5 ERR swss#orchagent: :- getQueueConfigurations: Invalid BUFFER_QUEUE key: [nfc421-5|Asic0|Ethernet176|0-2]

File: orchagent/portsorch.cpp (hunk 1 above)
Comment (skbarista) [reply orchagent/portsorch.cpp:6167]: @vmittal-msft https://github.com/sonic-net/sonic-swss/blob/f66abeda6a6e67b40b997b779ec3a6a362632fc1/orchagent/flexcounterorch.cpp#L331 still needs to enhanced to parse config for sysports. Till them we cannot conditionally enable voq counters.

**Comment by skbarista** [comment]
@vmittal-msft these are because getQueueConfigurations does not have code yet to parse voq config as I mentioned in my previous comment.

**Comment by skbarista** [comment]
@vmittal-msft just to add the getQueueConfigurations are not related to changes I have made.

File: orchagent/portsorch.cpp (hunk 1 above)
Comment (vmittal-msft) [reply orchagent/portsorch.cpp:6167]: One clarification, Will parsing fix egress queue counter fetching or VoQ counters ? Based on this, we may/may not revert your "always enable" logic.

**Comment by vmittal-msft** [comment]
What about these errors ? they seems to be new to me.

> Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x200051500000001 does not has supported counters
> Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x201051500000001 does not has supported counters
> Feb 16 16:37:50.544500 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x202051500000001 does not has supported counters
> Feb 16 16:37:50.544544 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x203051500000001 does not has supported counters
> Feb 16 16:37:50.544633 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x204051500000001 does not has supported counters

**Comment by skbarista** [comment]
> What about these errors ? they seems to be new to me.
> 
> > Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x200051500000001 does not has supported counters
> > Feb 16 16:37:50.544450 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x201051500000001 does not has supported counters
> > Feb 16 16:37:50.544500 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x202051500000001 does not has supported counters
> > Feb 16 16:37:50.544544 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x203051500000001 does not has supported counters
> > Feb 16 16:37:50.544633 nfc421-5 NOTICE syncd#syncd: :- addObject: Queue Counter oid:0x204051500000001 does not has supported counters

@vmittal-msft Let me check these and come back

**Review by vmittal-msft — APPROVED** [review]

**Pull request merged**